src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -806,6 +806,8 @@ prototype_file_noApplicableFileGroup = No Applicable Filegroup
 ############################################################################
 # Backup Service
 Backup_TaskName = Backup Database
+Backup_ServerCertificate = ServerCertificate;
```

Comment:
>ServerCertificate [](start = 27, length = 17)

Are these the correct string values?  I'd expect at least the words to have spaces between them for text we show to users..i.e., .these strings don't appear to require localization.